### DISCUSSION TITLE
[Config Support]: Live view grid size for 3:4 camera

### DISCUSSION THREAD

**Discussion by lin-xianming** [blakeblackshear/frigate#15277] [Config Support] [answered]
### Describe the problem you are having

The 3:4 camera takes up twice the grid height as the 16:9 cameras. Is there a way to make it the same size as the 16:9 cameras?
![frigate live grid](https://github.com/user-attachments/assets/1f14f2ac-01bb-43a2-9630-cb4819aed751)

### Version

0.14.1-f4f3cfa

### Frigate config file

```yaml
.
```

### Relevant Frigate log output

```shell
.
```

### Relevant go2rtc log output

```shell
.
```

### Operating system

Other Linux

### Install method

Docker Compose

### docker-compose file or Docker CLI command

```yaml
.
```

### Object Detector

OpenVino

**Comment by NickM-27** [comment]
Create a camera group and edit the layout

**Reply by lin-xianming** [reply]
It seems like the aspect ratio is fixed and I cannot make it the same size as the 16:9 cameras
![frigate live grid 2](https://github.com/user-attachments/assets/bc107f87-e23b-4a6d-b1ab-33708898ef76)

**Reply by NickM-27** [reply]
these look the same size to me

**Reply by lin-xianming** [reply]
I dragged it to the same size by holding the left mouse button and moving the mouse. Then I took this screenshot. Then when I release the left mouse button the size becomes the size of the red box. Which is half the size of the 16:9 cameras.

**Reply by NickM-27** [reply]
without having an actual visual example it is difficult to understand what you mean. As far as I can tell, there are 4 equal size 16:9 rectangles in the image

**Reply by lin-xianming** [reply]
This is all the possible sizes from half as wide to twice as tall. It is not possible to drag it to the same size at the 16:9 cameras and have it stay that size after stop dragging.
![frigate live grid 5](https://github.com/user-attachments/assets/55201740-5099-4829-95e5-666a7a0bda8c)
![frigate live grid 4](https://github.com/user-attachments/assets/ba841eca-e48f-4b9b-b4f0-4a8761c4041f)
![frigate live grid 3](https://github.com/user-attachments/assets/173462c9-9a7f-4922-86e6-52d9303d3c42)

**Comment by lin-xianming** [comment, marked as answer]
Not possible but was able to rebuild frigate with changes required.
